Modules/_tracemalloc.c

```diff
@@ -78,7 +78,10 @@ __attribute__((packed))
 
 typedef struct {
     Py_uhash_t hash;
+    /* Number of frames stored */
     int nframe;
+    /* Total number of frames the traceback had */
+    int length;
```

Comment:
Hum, maybe rename it to "total" or "total_nframe"?

Reply:
Let's go for `total_nframe`.